deployment-template/ec2/data-prepper-ec2-deployment-cfn.yaml

```diff
@@ -13,6 +13,9 @@ Parameters:
     AllowedPattern: "[a-z]+-[a-z]+-[0-9]+"
     Default: us-east-1
     ConstraintDescription: must be a valid AWS region (e.g. us-west-2)
+  AWSElasticsearchSubnetId:
+    Description: The subnet ID of the AWS Elasticsearch Service domain (Leave blank if the domain is not in a VPC)
```

Comment:
Could also add an e.g. as above

Reply:
Unfortunately subnets can be named whatever (the only constraint is a 256 character limit), so we can't really offer a meaningful example

Reply:
I was thinking some generic pattern like `subnet-4b18e22d(172.31.0.0/20)|us-east-1b` from the dropdown menu. But not sure if it is the subnet ID. Just to give cx a manifestation.